### PR TITLE
Check to see if there is data before showing the heading in a cell

### PR DESCRIPTION
**Pull request by babbeljames**

### CODE REVIEW

**Review by rnegash — APPROVED** [review]
🖼️ 🫒 🆕

File: src/components/CareerFramework.vue [hunk 1]
```diff
@@ -27,16 +27,20 @@
...
                 class="level"
               >
-                <b>{{ level.level }}</b
-                ><br />
-                <ul v-for="criteria in level.criteria" :key="criteria">
-                  <li>
-                    {{ criteria }}
-                  </li>
-                </ul>
+                <div v-if="level.criteria.length">
```
Comment (rnegash) [inline src/components/CareerFramework.vue:30]:
```suggestion
                <div v-if="level.criteria.length">
```
What happened if you put the `v-if` on the surrounding `td` element?

Comment (babbeljames) [reply src/components/CareerFramework.vue:30]: I think then it won't render the TD which will probably give us problems in a situation where there is an empty column somewhere in the middle of the row 🤔

Comment (rnegash) [reply src/components/CareerFramework.vue:30]: yeah makes sense

**Pull request merged**